BolajiOlajide/a_socials
# Ft populate user timezone 165057777

#### What Does This PR Do?
This PR populate the user's default timezone in event form

#### Description Of Task To Be Completed
- To populate timezone based on user location in the create event form

#### How can this be manually tested?
- Ensure docker is installed locally on machine
- Ensure the environment variables in .env.sample are added in .env file in the root directory
- To build and spin up the docker containers, run make build
- To spin up the containers at a different time after building, `run make start`
- Click on the create event button and check the timezone field

#### What are the relevant github issues?
[#165057777](https://www.pivotaltracker.com/story/show/165057777)

#### Screenshot(s)
<img width="632" alt="m" src="https://user-images.githubusercontent.com/29021978/55581200-29936a00-5714-11e9-807c-6dcc95408653.png">
